frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -714,9 +721,8 @@ export const supportLogic = kea<supportLogicType>([
                             browser: isFirefox ? 'firefox' : 'other',
                         }
                         posthog.capture('support_ticket', properties)
-                        lemonToast.success(
-                            "Got the message! If we have follow-up information for you, we'll reply via email."
-                        )
+                        lemonToast.success("Got the message! We'll reply to your ticket via email soon.")
+
```

Comment:
just some opportunistic updates as we'll always try to respond to a ticket, so users should expect a response!